### DISCUSSION TITLE
Request: Kool project template repo

### DISCUSSION THREAD

**Discussion by antzGames** [fabmax/kool#53] [Ideas]
So it was absolutely easy to build and run both desktop (jvm) and Web (Javascript) demos from this repo.  Fantastic!

I am wondering if you can quickly publish a repo that has a sample `Hello World` multiplatform project that uses **Kool as a library** as you described in the README.md. 

This would allow a newbie like me to start using Kool without the struggling getting gradle, plugins, versions, etc., figured out.

**Comment by fabmax** [comment]
You are right, I already had something similar in mind for quite some time 😄 

Anyway, here it is:
https://github.com/fabmax/kool-templates

**Comment by antzGames** [comment]
You are the best.  Thank you for this!
